modules/rte_mis_gin/js/about-us.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal, once) {
+    Drupal.behaviors.wrapBlockMenu = {
+        attach: function (context, settings) {
+            $(once('wrapAll', '.content-wrapper .menu--social-links, .content-wrapper .menu--get-in-touch', context)).wrapAll('<div class="menu-social-links-wrapper block"></div>');
+            $(once('wrapAll', '.content-wrapper .block-system', context)).wrapAll('<div class="layout-container"></div>');
+            var $footerItems = $(once('wrapAll', '.content-wrapper .footer__site-logo-wrapper, .content-wrapper .menu--quick-links, .content-wrapper .menu--support, .content-wrapper .menu-social-links-wrapper, .content-wrapper .block-site__menu-text-section-block'), context);
+            $footerItems.wrapAll("<div class='footer-wrapper'></div>");
+            $(once('wrapAll', '.region-subheader-wrapper .block-site-logo-section-block, .region-subheader-wrapper .menu--main-menu, .region-subheader-wrapper .menu--account', context)).wrapAll('<div class="main-menu-wrapper"></div>');
```

Comment:
Why we are doing this via JS?

Reply:
For creating a wrapper on some items from different regions, becuase we can't create any new regions.

Reply:
I can see that we are repeating this code in all the JS file, Can't we create a global JS for these things?